roles/setup_pemagent/defaults/main.yml

```diff
@@ -14,6 +14,8 @@ pg_pem_agent_password: ""
 pg_ssl: true
 disable_logging: true
 
+node_ssh_to_pemserver: true
+enable_pemagent_barman: true
```

Comment:
I don't think this variable name makes sense.  PEM is EDB software and Barman is open source.  Perhaps it makes more sense to have a variable for each?